tests/network/vmi_networking.go

```diff
@@ -725,17 +724,33 @@ var _ = SIGDescribe("[Serial][rfe_id:694][crit:medium][vendor:cnv-qe@redhat.com]
 				ipAddr := gatewayIPFromCIDR(networkCIDR)
 				Expect(libnet.PingFromVMConsole(serverVMI, ipAddr)).To(Succeed())
 
-				By("Checking ping (IPv4) to google")
-				Expect(libnet.PingFromVMConsole(serverVMI, "8.8.8.8", "-c 5", "-w 15")).To(Succeed())
-				Expect(libnet.PingFromVMConsole(clientVMI, "google.com", "-c 5", "-w 15")).To(Succeed())
-
 				Expect(verifyClientServerConnectivity(clientVMI, serverVMI, tcpPort, k8sv1.IPv4Protocol)).To(Succeed())
 			},
 				table.Entry("with a specific port number [IPv4]", []v1.Port{{Name: "http", Port: 8080}}, ""),
 				table.Entry("without a specific port number [IPv4]", []v1.Port{}, ""),
 				table.Entry("with custom CIDR [IPv4]", []v1.Port{}, "10.10.10.0/24"),
 			)
 
+			It("[outside_connectivity]should be able to reach the outside world [IPv4]", func() {
+				ipv4Address := "8.8.8.8"
+				if flags.IPV4ConnectivityCheckAddress != "" {
+					ipv4Address = flags.IPV4ConnectivityCheckAddress
+				}
+				dns := "google.com"
```

Comment:
google.com appears twice, consider using a const in the tests block Context please

Reply:
google is used once in vmi_networking and once in vmi_slirp.
I think it is better to not define a const to be used over those two tests files. It could just cause confusion when someone will work on the two files independently I think.